src/lib/merkle_mask/masking_merkle_tree.ml

```diff
@@ -125,47 +125,47 @@ struct
     let get_uuid t = assert_is_attached t ; t.uuid
 
     (* don't rely on a particular implementation *)
-    let find_hash t address =
+    let self_find_hash t address =
```

Comment:
yep, this helps make it clear that these operations work just on the mask tables.